src/tests/ftest/io/dfuse_find_cmd.yaml

```diff
@@ -22,8 +22,10 @@ container:
   control_method: daos
   cont_count: 5
 find_cmd:
-  samples: 3
   height: 3
   subdirs_per_node: 6
   files_per_node: 10
   needles: 100
+perf:
+  samples: 3
+  challenger_path: /mnt/lustre
```

Comment:
Well, does it run over Lustre by default ?

Reply:
The perf test is disabled :p
`@skipForTicket("DAOS-6530")`

It was suggested to compare DAOS against Lustre. AFAIK we don't have Lustre support on CI. So, I implemented the perf test and wrote "/mnt/lustre" path as an example. DAOS-6530 will enable the perf test.